src/pages/Settings/index.tsx

```diff
@@ -1,22 +1,105 @@
-import { IonHeader, IonPage, IonTitle, IonToolbar, IonText } from '@ionic/react'
-import { useNodes } from '../../util/ipfs'
-import PageContainer from '../../components/PageContainer'
-import './index.scss'
+import {
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonImg,
+  IonLabel,
+  IonRadioGroup,
+  IonItem,
+  IonRadio,
+} from "@ionic/react"
+import { useNodes } from "../../util/ipfs"
+import PageContainer from "../../components/PageContainer"
+import createPersistedState from "use-persisted-state"
+
+import "./index.scss"
+import ShortcutLinks from "../../components/Shortcuts"
+
+const gateways = [
```

Comment:
Can you pull this from the main nodes code at runtime, so this doesn't need to be hardcoded and updated every time the list in that file changes?